notebooks/dataset_tutorial/sensorium/utility/measure_helpers.py

```diff
@@ -1,49 +1,63 @@
-import pandas as pd
 import numpy as np
+import pandas as pd
 
 
-def get_df_for_scores(session_dict, measure_attribute='score'):
+def get_df_for_scores(session_dict, measure_attribute="score"):
     data_keys, values = [], []
     for key, unit_array in session_dict.items():
         for value in unit_array:
             data_keys.append(key)
             values.append(value)
-    df = pd.DataFrame({'dataset':data_keys, measure_attribute:values})
+    df = pd.DataFrame({"dataset": data_keys, measure_attribute: values})
     return df
 
 
 # Helpers for Readout-Position Color Plot
 
+
 # smooth color interpolation
 def lerp(x, a, b):
-    return a + x * (b-a)
+    return a + x * (b - a)
+
 
 # smooth color interpolation
 def serp(x, a, b):
```

Comment:
@Nisone2000 I would inspire you to name the functions a bit more explicit as I cannot guess what it is supposed to do from the name:)